Server/Source/AudioWorker.hpp

```diff
@@ -103,8 +103,14 @@ class AudioWorker : public Thread, public LogTagDelegate {
     bool waitForData();
 
     template <typename T>
-    AudioBuffer<T>* getProcBuffer();
-
+    AudioBuffer<T>* getProcBuffer()
+    {
+ 		if constexpr (std::is_same<T, float>::value) 
```

Comment:
The codebase is on C++14 currently.

Reply:
Thanks for reviewing the code. Sorry for the late reply, I am not much familiar with the process.  
My intention was to make the code compilable first, so that the other issues on linux can be fixed. That was the first step of the changes. The dummy functionality of CPU load and other things will be fixed later. But  found that basic functionality of audio flowing, plugin menu etc is working fine. 

About the review points:

FIXED    "don't build the server per default on linux."  (removed the part which trats server build on linux specially)  
FIXED    "run the code formatter (package/format.sh)" (ran the script and committed the changes)

    "resolve the compiler warnings (I have seen some sign-conversion and c++17 warnings)"
The compiler error in AudioWorker.hpp was because the issue in C++14  in this special case of template specialization. The fix i got on  StackOverflow and the same mentioned that the issue resolved in C++17. I will try to fix other warnings about unused variables and conversions etc.

Since I dont have proper setup, request you to please download the binary of server in release section of the branch and test it. 

Audio is fine. plugin list (dummy) for client menu was also fine. Test the screen capturing, vst plugin loading etc. May be you need to change the settings manually in cfg files as it doesn't show the tray icon . Priority of development  will be for scanning and loading plugins.  

Thanks
Durgesh Kumar